app/views/static_pages/community_expectations.html.erb

```diff
@@ -1,11 +1,87 @@
 <%= title('Community Expectations') %>
-<% filename = 'app/views/static_pages/dashboard_steps/community_expectations.md' %>
+<div class="gradient py-8">
+  <div class="col-xl-8 offset-xl-2 text-gray-500">
```

Comment:
These classes will apply the same styling we use for content elsewhere to all the elements within the div. So we can remove all the classes from the headings, paragraphs and lists and not have to maintain individual styling for those elements 🎉  

```suggestion
  <div class="lesson-content prose prose-lg prose-gold max-w-prose mx-auto">
```